src/lm_buddy/buddy.py

```diff
@@ -0,0 +1,41 @@
+from lm_buddy.integrations.wandb import ArtifactLoader, WandbArtifactLoader
+from lm_buddy.jobs._entrypoints import run_finetuning, run_lm_harness, run_prometheus, run_simple
+from lm_buddy.jobs.common import EvaluationResult, FinetuningResult, SimpleResult
+from lm_buddy.jobs.configs import (
+    EvaluationJobConfig,
+    FinetuningJobConfig,
+    LMHarnessJobConfig,
+    PrometheusJobConfig,
+    SimpleJobConfig,
+)
+
+
+class LMBuddy:
+    """Your buddy in the (L)LM space.
+
+    Simple wrapper around executable functions for tasks available in the library.
+    """
+
+    def __init__(self, artifact_loader: ArtifactLoader = WandbArtifactLoader()):
+        self._artifact_loader = artifact_loader
+
+    def simple(self, config: SimpleJobConfig) -> SimpleResult:
+        """Run a simple test task."""
+        return run_simple(config)
+
+    def finetune(self, config: FinetuningJobConfig) -> FinetuningResult:
+        """Run a supervised finetuning task with the provided configuration."""
+        return run_finetuning(config, self._artifact_loader)
```

Comment:
This is kind of confusing because we run_finetuning but we return results from run_finetuning, could be as simple as assigning it for legibility?

```python
def finetune(self, config: FinetuningJobConfig) -> FinetuningResult:
        """Run a supervised finetuning task with the provided configuration."""
        finetuning_results = run_finetuning(config, self._artifact_loader)
        return finetuning_results
```